src/markdown-pages/docs/first-steps/prerequisites.md

```diff
@@ -0,0 +1,71 @@
+---
+path: "/docs/prerequisites"
```

Comment:
I can't see this page in the contents, or any way to get to it.

Reply:
ah yes I put the pre-reqs in each section as they were slightly different so it made better sense to separate them. I did not delete this file though so have done that now.